src/OSPSuite.R/Services/DataImporterTask.cs

```diff
@@ -21,6 +25,18 @@ public interface IDataImporterTask
       IReadOnlyList<DataRepository> ImportXslxFromConfiguration(ImporterConfiguration configuration, string dataFileName);
       IReadOnlyList<DataRepository> ImportCsvFromConfiguration(string configurationFileName, string dataFileName, char columnSeparator);
```

Comment:
is dataFileName a Path? If yes, should not we call it Path of fullPath or sthg like this?